001-core/privacy.php

```diff
@@ -60,6 +60,7 @@ function generate_personal_data_export_file( $request_id ) {
 	// Create the exports folder if needed.
 	$exports_dir = wp_privacy_exports_dir();
 	$exports_url = wp_privacy_exports_url();
+	$temp_dir = get_temp_dir();
 
 	$result = wp_mkdir_p( $exports_dir );
```

Comment:
Is this step still necessary? Or even the `$exports_dir` variable itself necessary?

Reply:
Yeah, still needed for sites not using stream wrappers (AMS).